### ISSUE TITLE
factoryName?

### ISSUE THREAD

**Issue by kentcdodds** [jackfranklin/test-data-bot#238]
What's the purpose of the `factoryName`? It isn't used anywhere internally and feels superfluous for the typical use case:

```javascript
const userBuilder = build('User', {
  fields: {
    name: 'jack',
  },
});
```

That's no more readable than:

```javascript
const userBuilder = build({
  fields: {
    name: 'jack',
  },
});
```

I'd like to request that the `factoryName` be optional (or simply removed).

**Comment by jackfranklin** [comment]
Agreed, I included it at first because I expected it to be useful for debugging but I've found no reason for it. I'm going to make it optional to avoid the breaking change.

**Comment by kentcdodds** [comment]
Thanks :)

**Comment by jackfranklin** [comment]
Released in 1.2
